pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-spark-3.2/src/main/java/org/apache/pinot/plugin/ingestion/batch/spark/SparkSegmentGenerationJobRunner.java

```diff
@@ -0,0 +1,408 @@
+/**
```

Comment:
Will there be a problem when both spark 2.4 and 3.2 jars are presented with the same package and class name?
Or do you intend to do this for job spec file simplicity?

Reply:
Renamed to spark3. Kept to existing one as `spark` only instead of `spark2` as it was touching a lot of classes.